HappyAnding/HappyAnding/Views/Components/UserCurationCell.swift

```diff
@@ -7,14 +7,140 @@
 
 import SwiftUI
 
+//MARK: - 테스트를 위한 모델 구현 (추후 삭제 필요)
```

Comment:
후행 작업이 있어서 TODO가 더 적절해보여요!

```suggestion 
// TODO: 테스트를 위한 모델 구현 (추후 삭제 필요)
```